### PR TITLE
Fix spelling in README

### PR DESCRIPTION
**Pull request by SirMorfield**
⚠️⚠️⚠️ Since we do not accept all types of pull requests and do not want to waste your time. Please be sure that you have read pull request rules:
https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#can-i-create-a-pull-request-for-uptime-kuma

Tick the checkbox if you understand [x]: 
- [x] I have read and understand the pull request rules.

# Description

Fix some spelling in the README
Add docker-compose to the README

## Type of change
- Translation update
- Other

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I ran ESLint and other linters for modified files
- [x] I have performed a self-review of my own code and tested it
- [x] I have commented my code, particularly in hard-to-understand areas
  (including JSDoc for methods)
- [x] My changes generate no new warnings
- [x] My code needed automated testing. I have added them (this is optional task)

### CODE REVIEW

**Comment by louislam** [comment]
No plan to add this in README.md. And it is already available in:
- https://github.com/louislam/uptime-kuma/blob/master/docker/docker-compose.yml
- https://github.com/louislam/uptime-kuma/wiki/%F0%9F%94%A7-How-to-Install

**Comment by SirMorfield** [comment]
@louislam See above comment

File: README.md [hunk 1]
```diff
@@ -44,14 +44,14 @@ docker run -d --restart=always -p 3001:3001 -v uptime-kuma:/app/data --name upti
...
-Browse to http://localhost:3001 after starting.
+Kuma is now running on http://localhost:3001
 
 ### 💪🏻 Non-Docker
 
 Required Tools: 
 - [Node.js](https://nodejs.org/en/download/) >= 14
 - [Git](https://git-scm.com/downloads) 
-- [pm2](https://pm2.keymetrics.io/) - For run in background
+- [pm2](https://pm2.keymetrics.io/) - For running kuma in the background
```
Comment (Saibamen) [inline README.md:54]:
```suggestion
- [pm2](https://pm2.keymetrics.io/) - For running Kuma in the background
```

**Comment by Saibamen** [comment]
Please rename PR title

File: README.md (hunk 1 above)
Comment (louislam) [reply README.md:54]: Should be `Uptime Kuma`

Comment (SirMorfield) [reply README.md]: Done

**Pull request merged**